lib/ui/screens/create_post/create_post_view.dart

```diff
@@ -1,15 +1,65 @@
 import 'package:flutter/material.dart';
 import 'package:fpdart/fpdart.dart';
+import 'package:medi_support/ui/screens/create_post/create_post_controller.dart';
 import 'package:medi_support/ui/widgets/custom_app_bar.dart';
+import 'package:medi_support/ui/widgets/custom_text_form_field.dart';
 
 class CreatePostView extends StatelessWidget {
-  const CreatePostView({super.key});
+  CreatePostView({super.key, required this.controller});
+  final CreatePostController controller;
+
+  final TextEditingController bodyController = TextEditingController();
+  final TextEditingController titleController = TextEditingController();
+
+  Widget _buildTitleField() => Builder(
+        builder: (BuildContext context) => CustomTextFormField(
+          hint: "title",
+          label: 'Title',
+          controller: titleController,
+        ),
+      );
+
+  Widget _buildBodyField() => Builder(
+        builder: (BuildContext context) => CustomTextFormField(
+          hint: "body",
+          label: 'Body',
+          controller: bodyController,
+          minLines: 5,
+          maxLines: 25,
+        ),
+      );
 
   @override
   Widget build(BuildContext context) => Scaffold(
         appBar: CustomAppBar(
           leading: some(
-            IconButton(onPressed: () {}, icon: const Icon(Icons.close)),
+            IconButton(
+              onPressed: controller.goHome,
+              icon: const Icon(Icons.close),
+            ),
+          ),
+          actions: <Widget>[
+            IconButton(
```

Comment:
**suggestion:** Should be an `ElevatedButton` or similar. You could also adjust the design.

Reply:
looks 5 times better if its an icon button